src/base/algodiff/owl_algodiff_ops_builder_sig.ml

```diff
@@ -109,4 +109,13 @@ module type Sig = sig
   end
 
   val build_piso : (module Piso) -> t -> t -> t
+
```

Comment:
This whole module should really be documented...

Reply:
NB: I am not suggesting to do it as part of this PR

Reply:
While becoming more flexible and powerful, Algodiff is definitely getting more complicated. It deserve a big chapter in the tutorial book, details its design and application :)

Reply:
yea definitely!